### DISCUSSION TITLE
Display recently modified posts in the calendar

### DISCUSSION THREAD

**Discussion by andergmartins** [publishpress/PublishPress-Planner#1302] [Feature Requests]
​Displaying posts based on the modification date and not only on the publishing date. 

From the client:
>Is there a hook that I can use to force the calendar to display updated posts in addition to those that were recently published? If not would you consider adding a setting to enable the display of recently changed content?

**Comment by alumnionlinewebservices-repo** [comment]
Having an option to see the changed pages, not just those that are published I feel is an important feature. Is there an expected implementation date for this enhancement?

**Comment by andergmartins** [comment]
Hi @seshelby sorry, not yet. No rough date for now.
